src/platform/PICO/include/platform/platform.h

```diff
@@ -88,6 +88,11 @@ typedef enum {DISABLE = 0, ENABLE = !DISABLE} FunctionalState;
 
 #define SCHEDULER_DELAY_LIMIT           10
 
+// There is no library definition for pupd, so define one here
+#define GPIO_PULLUP     1
+#define GPIO_PULLDOWN   2
```

Comment:
Are they not defined in PICO library / middleware ?

Reply:
No